.nvmrc

```diff
@@ -0,0 +1 @@
+18.15.0
```

Comment:
We might want to update the README, it currently says we use Node 16.

Reply:
good point!